pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -546,7 +546,7 @@ tls_cert_path {{ .SecretPath "ca-bundle.crt"}}
 <buffer>
   @type file
   path '{{.BufferPath}}'
-  queued_chunks_limit_size "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
+  queue_limit_length "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
```

Comment:
What should be the correct fallback here?  In one place we say 32 and the other 1024?  seems like the default is maybe conflating two distinct concepts?

Reply:
You need to help me out here. This is the buffer for secure-forward/forward.

Reply:
AFAICS per git blame, this was introduced with LogForwarding.

Reply:
It's a hold over from earlier releases.

Reply:
I think 32 is probably the "correct" value since its a number of files is it not?

Reply:
Yes queue_limit_length refers to number of chunk files.